packages/ui/src/stories/Text.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import { Meta } from '@storybook/react'
```

Comment:
I think we are missing here this part
https://github.com/react-to/react-to-styled/blob/develop/.storybook/preview.js
Where you will import GlobalStyle to the storybook?

Reply:
@aleksaToljic  Now I see :D Fixed align and imported GlobalStyle to storybook